### DISCUSSION TITLE
Questions in this repository

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3280] [General] [answered]
Now that [discussions](https://github.com/corona-warn-app/cwa-website/discussions) have been enabled in this repository, should all questions now be placed here instead of in [issues](https://github.com/corona-warn-app/cwa-website/issues)?

If yes, then existing open [questions](https://github.com/corona-warn-app/cwa-website/labels/question) should be moved to [discussions](https://github.com/corona-warn-app/cwa-website/discussions).

Also the templates in https://github.com/corona-warn-app/cwa-website/tree/master/.github/ISSUE_TEMPLATE would need to be modified to point questions to [discussions](https://github.com/corona-warn-app/cwa-website/discussions).

**Comment by brianebeling** [comment, marked as answer]
#2102 is already mirrored and seems actionable. So I left it as an issue. I will rename the title to something more fitting though.
I will create an issue for the ISSUE_TEMPLATE update. Thanks!

The other issues have been converted.
